PlateSpace/Web/README.md

```diff
@@ -35,11 +35,11 @@ In order to get our Stitch backend-as-a-service application fully set up, we mus
 ### Named pipelines
 
 Stitch lets you define named pipelines.
-Named pipelines are custom pipeline requests that could be executed from the client side / by rules on top of the collections (see Rules section below). To create a named pipeline, select the  `Pipelines` link in the left-hand menu and press the `New` button close to the `Named Pipelines` text.
+Named pipelines are custom pipeline requests that could be executed from the client side or by collection (see [Rules configuration](https://github.com/mongodb/stitch-examples/tree/master/PlateSpace/Web#rules-configuration) below). To create a named pipeline, select the  `Pipelines` link in the left-hand menu and press the `New` button close to the `Named Pipelines` text.
```

Comment:
fixed in https://github.com/mongodb/stitch-examples/pull/15/commits/a83b6c79275ffffae282ab74153c3f8bfc037cfd